manifests/h/Hashicorp/Vagrant/2.3.3/Hashicorp.Vagrant.installer.yaml

```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Hashicorp.Vagrant
+PackageVersion: 2.3.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /norestart
+InstallerSuccessCodes:
+- 3010
+- 1706
+UpgradeBehavior: install
+InstallLocationRequired: true
+RequireExplicitUpgrade: true

```

Comment:
```suggestion
```
This field is set for packages that auto-update themselves. Setting it to `true` excludes it from being upgraded through `winget upgrade --all`. Is it the case with this package?

Reply:
No that's a mistake